File: BlackJack.ViewModels/PlayerViews/GetAllPlayerView.cs
using System.Collections.Generic;

namespace BlackJack.ViewModels.PlayerViews
{
    public class GetAllPlayerView
    {
        public List<PlayerGetAllPlayerViewItem> Players { get; set; }

        public GetAllPlayerView()
        {
            Players = new List<PlayerGetAllPlayerViewItem>();
        }
    }

    public class PlayerGetAllPlayerViewItem
    {
        public string PlayerId { get; set; }
        public string UserName { get; set; }
        public decimal Balance { get; set; }
        public decimal Bet { get; set; }
    }
}

File: BlackJack.WEB/ClientApp/src/app/shared/entities/player.views/get-all.player.view.ts
export class GetAllPlayerView {
  players?: Array<PlayerGetAllPlayerViewItem>;

  constructor() {
    this.players = new Array<PlayerGetAllPlayerViewItem>();
  }
}

export class PlayerGetAllPlayerViewItem {
  userName?: string;
  id?: string;
  balance?: number;
  bet?: number;
}
